sdk/src/main/java/com/mapbox/maps/renderer/RenderHandlerThread.kt

```diff
@@ -11,7 +11,6 @@ internal class RenderHandlerThread {
 
   @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
   internal lateinit var handlerThread: HandlerThread
-  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
   internal var handler: Handler? = null
```

Comment:
```suggestion
  private var handler: Handler? = null
```
?

Reply:
It is accessed from map controller, so can't be private